scripts/publish-site.js

```diff
@@ -37,26 +36,18 @@ removeSync(GENERATED_SITE_DIRECTORY);
 console.log('Generating site...');
 execSync(`bundle exec jekyll build --source ${SITE_DIRECTORY} --destination ${GENERATED_SITE_DIRECTORY}`, { cwd: SITE_DIRECTORY, stdio: 'ignore' });
 
-console.log('Coping github staff...');
+console.log('Copying github staff...');
 copySync(join(__dirname, 'gh-pages-files'), GENERATED_SITE_DIRECTORY);
 
 console.log('Publishing...');
-publish(GENERATED_SITE_DIRECTORY, {
-  branch: BRANCH,
-  repo: `https://github.com/${REPO}.git`,
-  message: COMMIT_MESSAGE,
-  src: ['**/*', '.gitattributes'],
-}, (err) => {
-  if (err) {
-    console.log(String(err));
-    return;
-  }
-
-  console.log();
-  console.log('--------------------');
-  console.log('Done!');
-  console.log();
-  console.log(`You have to check that everething is good at https://devexpress.github.io/devextreme-reactive/`);
-  console.log('--------------------');
-  console.log();
-});
+execSync(`git add -f ${GENERATED_SITE_DIRECTORY}`);
+execSync(`git commit -m "${COMMIT_MESSAGE}"`);
+execSync(`git subtree push --prefix ${GENERATED_SITE_DIRECTORY} https://github.com/${REPO}.git ${BRANCH}`);
```

Comment:
Why do you use repo link instead of `upstream` here? I am not against that but `upstream` is used in the 'scripts/ensure-repo-up-to-date.js' script. Can we use unified 'upstream' access in scripts?